open-api/NOTICE

```diff
@@ -81,6 +81,9 @@ Copyright 2002-2024 The Apache Software Foundation
 Apache Commons Lang
 Copyright 2001-2023 The Apache Software Foundation
 
+Apache Commons Configuration
```

Comment:
Are you sure about this ? 
I checked the `NOTICE` file in commons-configuration, and I see:

```
Apache Commons Configuration
Copyright 2001-2013 The Apache Software Foundation

This product includes software developed at
The Apache Software Foundation (http://www.apache.org/).
```

As reminder, the purpose of ALv2 4.d, is to include `NOTICE` content (as it is) in the `NOTICE` file.

Reply:
I am sure. 

https://github.com/apache/commons-configuration/blob/master/NOTICE.txt

> This product includes software developed at
The Apache Software Foundation (http://www.apache.org/).

This line is there at the beginning of the section

Reply:
I added this for each one instead of section. So, notice looks as it is.